src/components/views/right_panel/BaseCard.tsx

```diff
@@ -81,43 +83,46 @@ const BaseCard: React.FC<IProps> = forwardRef<HTMLDivElement, IProps>(
     ) => {
         let backButton;
         const cardHistory = RightPanelStore.instance.roomPhaseHistory;
-        if (cardHistory.length > 1) {
+        if (cardHistory.length > 1 && !hideHeaderButtons) {
             const prevCard = cardHistory[cardHistory.length - 2];
-            const onBackClick = (ev: ButtonEvent): void => {
+            const onBackClick = (ev: MouseEvent<HTMLButtonElement>): void => {
                 onBack?.(ev);
                 RightPanelStore.instance.popCard();
             };
             const label = backLabelForPhase(prevCard.phase) ?? _t("action|back");
-            backButton = <AccessibleButton className="mx_BaseCard_back" onClick={onBackClick} title={label} />;
+            backButton = (
+                <IconButton
+                    size="28px"
+                    data-testid="base-card-back-button"
+                    onClick={onBackClick}
+                    tooltip={label}
+                    subtleBackground
+                >
+                    <ChevronLeftIcon />
```

Comment:
The tooltip doesn't seem to function as an accessible label on these currently

![Screenshot from 2024-07-12 17-34-56](https://github.com/user-attachments/assets/aac5559c-bfa5-4936-8978-f3a2c80cb66b)

Reply:
Compound Tooltips' accessible labels are only added as an aria-labelledby/aria-describedby on focus

![image](https://github.com/user-attachments/assets/b149e3d4-5821-4491-aa06-65a07163fd41)
![image](https://github.com/user-attachments/assets/d585798f-2e5f-4185-bc90-8578c2756822)

Reply:
I would think that an icon like this should still have a label or description even when unfocused, right? Or is that assumption wrong? (like, do assistive technologies generally never try to describe things that aren't in focus, or is that a design decision specific to certain screen readers?)

Reply:
Yes it would be preferable if anything with a tooltip would have an aria-label rather than relying on `aria-*by` - though that'd be on the part of Compound to do, otherwise both the label & description would be the same string and cause double-reading.

Reply:
Okay, so if I combine this with a little advice from https://sarahmhigley.com/writing/tooltips-in-wcag-21/ … what I'm getting is that Compound tooltips could do with a 'label' mode which they provide `aria-labeledby` on the target, stay in the DOM, and are not `aria-hidden` but merely visually hidden. Or, something of that nature.

If you're comfortable with merging this before Compound tooltips have gone through that iteration, I don't mean to keep this blocked.

Reply:
Yes that would be preferable on the Compound side, but not a blocker here given pretty much all tooltips in Element Web already use Compound tooltips

Reply:
Right, I'm just mindful of the fact that most icons with tooltips in the app have opted to go the double-reading route for now.

Reply:
![image](https://github.com/user-attachments/assets/65764e40-2c7b-42a4-a555-ad7242298fa2)

Have they?

Reply:
That's my understanding, see for example the duplicate `label` and `aria-label` strings in the screenshot

Reply:
Oh yes I missed it in my own screenshot, duh. Well that's a horrible bug in Compound

Reply:
Have opened https://github.com/element-hq/compound/issues/322